diff -r 5f0e3805cbe4 src/cpu/x86/vm/templateInterpreterGenerator_x86_64.cpp
--- a/src/cpu/x86/vm/templateInterpreterGenerator_x86_64.cpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/cpu/x86/vm/templateInterpreterGenerator_x86_64.cpp	Wed Sep 07 03:03:18 2016 +0200
@@ -352,7 +352,7 @@
   // stack: [ ret adr ] <-- rsp
 
   address entry_point = __ pc();
-  const char *msg = "\x0d0\x09f\x0d1\x080\x0d0\x0b8\x0d0\x0b2\x0d0\x0b5\x0d1\x082 JBreak!\n";
+  const char *msg = "Hello JavaZone!\n";
   __ mov64(c_rarg1, (long)stdout);
   __ mov64(c_rarg0, (long)msg);
   __ xorl(rax, rax);
diff -r 5f0e3805cbe4 src/share/vm/classfile/vmSymbols.hpp
--- a/src/share/vm/classfile/vmSymbols.hpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/share/vm/classfile/vmSymbols.hpp	Wed Sep 07 03:03:18 2016 +0200
@@ -1355,7 +1355,7 @@
    do_name(     getAndSetObject_name,                                   "getAndSetObject")                                    \
    do_signature(getAndSetObject_signature,                              "(Ljava/lang/Object;JLjava/lang/Object;)Ljava/lang/Object;" ) \
                                                                                                                                \
-  /* support for JBreak */                                                                                                     \
+  /* support for JavaZone */                                                                                                   \
   do_class(     java_util_Random,         "java/util/Random")                                                                  \
   do_intrinsic(_nextInt,                  java_util_Random,             nextInt_name, void_int_signature, F_R)                 \
    do_name(     nextInt_name,                                           "nextInt")                                             \
diff -r 5f0e3805cbe4 src/share/vm/interpreter/abstractInterpreter.cpp
--- a/src/share/vm/interpreter/abstractInterpreter.cpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/share/vm/interpreter/abstractInterpreter.cpp	Wed Sep 07 03:03:18 2016 +0200
@@ -147,7 +147,7 @@
   case vmIntrinsics::_doubleToRawLongBits: return java_lang_Double_doubleToRawLongBits;
   }
 
-  if (JBreak) {
+  if (JavaZone) {
     if (m->intrinsic_id() == vmIntrinsics::_sayHello) {
       return HelloWorld_sayHello;
     }
diff -r 5f0e3805cbe4 src/share/vm/interpreter/templateInterpreterGenerator.cpp
--- a/src/share/vm/interpreter/templateInterpreterGenerator.cpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/share/vm/interpreter/templateInterpreterGenerator.cpp	Wed Sep 07 03:03:18 2016 +0200
@@ -265,7 +265,7 @@
       method_entry(java_lang_Double_longBitsToDouble);
       method_entry(java_lang_Double_doubleToRawLongBits);
 
-      if (JBreak) {
+      if (JavaZone) {
         method_entry(HelloWorld_sayHello);
       }
 #undef method_entry
diff -r 5f0e3805cbe4 src/share/vm/oops/method.cpp
--- a/src/share/vm/oops/method.cpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/share/vm/oops/method.cpp	Wed Sep 07 03:03:18 2016 +0200
@@ -1393,7 +1393,7 @@
   // exception: the AES intrinsics come from lib/ext/sunjce_provider.jar
   // which does not use the class default class loader so we check for its loader here
   const InstanceKlass* ik = InstanceKlass::cast(holder);
-  if (JBreak && ik->name()->equals("org/simonis/HelloWorld")) {
+  if (JavaZone && ik->name()->equals("org/simonis/HelloWorld")) {
     // check for org.simonis.HelloWorld
     return vmSymbols::find_sid(ik->name());
   }
diff -r 5f0e3805cbe4 src/share/vm/runtime/globals.hpp
--- a/src/share/vm/runtime/globals.hpp	Wed Sep 07 02:57:30 2016 +0200
+++ b/src/share/vm/runtime/globals.hpp	Wed Sep 07 03:03:18 2016 +0200
@@ -4044,8 +4044,8 @@
   diagnostic(int,  CompilerDirectivesLimit, 50,                             \
              "Limit on number of compiler directives.")                     \
                                                                             \
-  product(bool, JBreak, false,                                              \
-          "Enable some JBreak specific enhancements")
+  product(bool, JavaZone, false,                                              \
+          "Enable some JavaZone specific enhancements")
 
 
 /*
